config.go

```diff
@@ -106,6 +108,27 @@ func (c *Config) Discover() error {
 		}
 	}
 
+	// Finally, we'll fill in any internal plugins that haven't already been
+	// discovered on disk.
```

Comment:
Ah yes that reminds me we're actually bumping the plugin API version in 0.7, so old plugins will definitely blow up w/ a version mismatch. We can play around with the code path and layer on some nicer messages explaining what to do. :+1: